packages/ai/core/generate-text/index.ts

```diff
@@ -21,15 +21,9 @@ export type {
   StreamTextResult,
   TextStreamPart,
 } from './stream-text-result';
-export type {
-  CoreToolCall,
-  CoreToolCallUnion,
-  ToolCall,
-  ToolCallUnion,
-} from './tool-call';
+export type { CoreToolCallUnion, ToolCall, ToolCallUnion } from './tool-call';
```

Comment:
rm `CoreToolCallUnion`?

Reply:
or was that in the other pr?

Reply:
yes that was removed in https://github.com/vercel/ai/pull/5768/files#diff-97e4c4e0b8b41041865409d1692cc8374efa62a36b8a50e736e799fb46829255L26, I'll merge `v5` into this branch as it caused conflicts. I tried to split up the changes by packages for easier reviews